packages/dotnet-sdk/src/TeamsFx/Bot/TeamsBotSsoPrompt.cs

```diff
@@ -77,54 +76,60 @@ public class TeamsBotSsoPrompt : Dialog
 {
     private readonly TeamsBotSsoPromptSettings _settings;
     private const string PersistedExpires = "expires";
+    internal IIdentityClientAdapter _identityClientAdapter { private get; set; }
```

Comment:
is this visible to developers?

Reply:
No. Internal types or members are accessible only within files in the same assembly. [Link](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/internal).
Or through `InternalsVisibleToAttribute` configuration. (In our code we use this attribute to expose the members to test project)